android/src/main/java/io/invertase/firebase/messaging/RNFirebaseMessaging.java

```diff
@@ -73,6 +74,18 @@ public void getToken(Promise promise) {
   }
 
   @ReactMethod
+  public void deleteInstanceId(Promise promise){
+    try {
+      Log.d(TAG, "Deleting instance id");
+      FirebaseInstanceId.getInstance().deleteInstanceId();
+      promise.resolve(null);
+    } catch (IOException e) {
+      e.printStackTrace();
```

Comment:
Can you change this to log out the exception using `Log` rather than printStackTrace?

Reply:
Done